ycms/cms/templates/patients_list.html

```diff
@@ -7,34 +7,181 @@
+                            </td>
+                            <td class="px-6 py-4 text-center">
+                                <span>
+                                    {% if patient.current_room %}
+                                        {% url 'cms:protected:ward_detail' pk=patient.current_ward.ward_number as url %}
+                                        Stays in: <a class="text-blue-500" href="{{ url }}">{{ patient.current_ward.name }}, Room {{ patient.current_room.room_number }}</a>
+                                    {% elif patient.current_stay %}
+                                        {% url 'cms:protected:ward_detail' pk=patient.current_stay.recommended_ward.ward_number as url %}
+                                        <b class="text-red-500">Waits for assignment
+                                            {% if patient.current_stay.recommended_ward %}
+                                                in Ward <a class="text-blue-500" href="{{ url }}">{{ patient.current_stay.recommended_ward.name }}</a>
+                                            {% endif %}
+                                        </b>
+                                    {% else %}
+                                        -
+                                    {% endif %}
+                                </span>
+                            </td>
+                            <td>
+                                <button class="btn" onclick="editPatient({{ patient.id }})">{% translate "Edit" %}</button>
+                                <button class="btn hidden" onclick="editPatient({{ patient.id }})">{% translate "Cancel" %}</button>
+                                <button class="btn hidden" form="delete-patient-{{ patient.id }}">
+                                    <form hidden id="delete-patient-{{ patient.id }}" method="post">
+                                        {% csrf_token %}
+                                        <input type="text" name="id" value="{{ patient.id }}" />
+                                    </form>
+                                    {% translate "Delete" %}
+                                </button>
+                                <button class="btn hidden" form="edit-patient-{{ patient.id }}">
+                                    <form hidden id="edit-patient-{{ patient.id }}" method="post">
+                                        {% csrf_token %}
+                                        <input type="text" name="id" value="{{ patient.id }}" />
+                                    </form>
+                                    {% translate "Save" %}
+                                </button>
+                            </td>
+                        </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+            <script>
+                // TODO: move to typescript file
+                function editPatient(id) {
```

Comment:
```suggestion
                const editPatient = (id: string) => {
```
same below, eslint will complain otherwise once you move this to its proper file